tests/integration/network/sepolia/conftest.py

```diff
@@ -15,6 +14,6 @@
 # run all tests with testnet
 @pytest.fixture(scope="module", autouse=True)
 def sepolia_env():
-    with boa.swap_env(NetworkEnv(SEPOLIA_URI)):
+    with boa.set_network_env(SEPOLIA_URI):
```

Comment:
does set_network_env give a context manager?

Reply:
reverted in https://github.com/vyperlang/titanoboa/pull/183/commits/e6c6d95ef762d4ad9b1b78f2e31e5aa940281da7